cuda/apps/src/kinect_viewer_cuda.cpp

```diff
@@ -77,7 +77,7 @@ class KinectViewerCuda
       pcl::Grabber* interface = new pcl::OpenNIGrabber(device_id);
 
       std::function<void (const boost::shared_ptr<openni_wrapper::Image>& image, const boost::shared_ptr<openni_wrapper::DepthImage>& depth_image, float)>
-        f = std::bind (&KinectViewerCuda::cloud_cb_, this, _1, _2, _3);
+        f = [&](const auto& image, const auto& depth_image, float constant) { cloud_cb_(image, depth_image, constant); };
```

Comment:
We decided a couple of PRs ago, that we would stick to explicitly typing the signatures. See https://github.com/PointCloudLibrary/pcl/pull/3136#discussion_r291530193

```suggestion
        f = [&](const openni_wrapper::Image::Ptr& image, const openni_wrapper::DepthImage::Ptr& depth_image, float constant) { cloud_cb_(image, depth_image, constant); };
```